db/migrations/00012_potgraphile_triggers.sql

```diff
@@ -0,0 +1,70 @@
+-- +goose Up
+-- +goose StatementBegin
+create function eth.graphql_subscription() returns trigger as $$
+declare
+    table_name text = TG_ARGV[0];
+    attribute text = TG_ARGV[1];
+    id text;
+begin
+    execute 'select $1.' || quote_ident(attribute)
+        using new
+        into id;
+    perform pg_notify('postgraphile:' || table_name,
+                      json_build_object(
+                              '__node__', json_build_array(
+                              table_name,
```

Comment:
Looks like this isn't always exactly the table name (supposed to base64 decode nodeId to see what it is), but I'm guessing it works out in our cases?